android/expoview/src/main/java/host/exp/exponent/utils/ScopedContext.kt

```diff
@@ -157,6 +167,10 @@ class ScopedContext @Throws(UnsupportedEncodingException::class) constructor(con
       filesDir = baseContext.filesDir
       cacheDir = baseContext.cacheDir
       noBackupDir = baseContext.noBackupFilesDir
+    } else {
+      listOf(filesDir, cacheDir, noBackupDir).forEach {
+        ensureDirExists(it)
+      }
```

Comment:
```suggestion

    listOf(filesDir, cacheDir, noBackupDir).forEach {
      ensureDirExists(it)
```

Reply:
nit: it doesn't hurt to call ensureDirExists for standalone apps. most importantly, it will benefit when we removing the standalone code, simply remove the `if (Constants.isStandaloneApp()) { ... }` block.